source/server/filter_chain_manager_impl.cc

```diff
@@ -23,6 +23,104 @@ Network::Address::InstanceConstSharedPtr fakeAddress() {
 
 } // namespace
 
+FilterChainFactoryContextImpl::FilterChainFactoryContextImpl(
+    Configuration::FactoryContext& parent_context, uint64_t filter_chain_tag)
+    : parent_context_(parent_context), filter_chain_tag_(filter_chain_tag) {}
+
+// FilterChainFactoryContext
```

Comment:
nit: not done in impl files, same elsewhere

Reply:
Err, I don't get it. Could you explain more?

Reply:
Sorry, this type of comment saying what interface it is. I would just delete.

Reply:
Get it. Thanks!